### PR TITLE
Replace non-ascii characters in headers to be in line with the service.

### PR DESCRIPTION
**Pull request by AsafMah**
### Fixed
* Replace non-ascii characters in headers to be in line with the service.

### CODE REVIEW

File: kusto/conn.go [hunk 1]
```diff
@@ -192,6 +193,21 @@ func (c *Conn) doRequestImpl(
 	headers http.Header,
 	errorContext string) (http.Header, io.ReadCloser, error) {
 
+	// Replace non-ascii chars in headers with '?'
+	for _, values := range headers {
+		var builder strings.Builder
+		for i := range values {
```
Comment (yogilad) [inline kusto/conn.go:199]: Q: How come values is a range on top of headers?

**Review by yogilad — APPROVED** [review]
Question in comments

File: kusto/conn.go (hunk 1 above)
Comment (AsafMah) [reply kusto/conn.go:199]: They made it so headers can be repeated, so each header is an array of strings (if that's what you meant)

Comment (yogilad) [reply kusto/conn.go:199]: Thanks!

**Pull request merged**